awx/main/tests/data/inventory/plugins/gce/files/gcp_compute.yml

```diff
@@ -0,0 +1,46 @@
+auth_kind: serviceaccount
+compose:
+  ansible_ssh_host: networkInterfaces | json_query("[0].accessConfigs[0].natIP")
```

Comment:
Alternatively use `networkInterfaces[0]['accessConfigs'][0]['natIP']` and negate the need of `json_query`

Reply:
It looks like I could go one step further, and be able to use `networkInterfaces[0].accessConfigs[0].natIP`

(have tested locally, and works for pretty much all cases we have here)

Since you're working on testing on the Ansible core side, I would like to go with your syntactical preference. Either of these 2 options are fine with me. Let me know which you want to go with.